letsencrypt/plugins/manual.py

```diff
@@ -153,12 +153,19 @@ def _perform_single(self, achall):
             ct=response.CONTENT_TYPE, port=port)
         if self.conf("test-mode"):
             logger.debug("Test mode. Executing the manual command: %s", command)
+            # sh shipped with OS X does't support echo -n
+            if sys.platform == "darwin":
+                executable = "/bin/bash"
+            else:
+                executable = "/bin/sh"
```

Comment:
I think this should be `executable = None` to make sure platform's default is used. https://docs.python.org/2/library/subprocess.html#popen-constructor